graphbolt/include/graphbolt/csc_sampling_graph.h

```diff
@@ -66,6 +66,29 @@ class CSCSamplingGraph : public torch::CustomClassHolder {
       const torch::optional<torch::Tensor>& node_type_offset,
       const torch::optional<torch::Tensor>& type_per_edge);
 
+  /**
+   * @brief Performs neighbor sampling for a given set of seed nodes taking edge
+   * type into account, where each edge type has a specified pick number.
+   *
+   * @param seed_nodes The tensor containing the seed nodes.
+   * @param fanouts The list containing the number of neighbors to sample.
```

Comment:
Why fanout is list?

Reply:
Tensor is pretty slow in different threads, so use list. For python interface ,it use tensor to keep consistent.